packages/turf-unkink-polygon/lib/simplepolygon.ts

```diff
@@ -100,19 +103,27 @@ function simplepolygon(feature) {
         })
       );
     }
-    var output = featureCollection(outputFeatureArray);
-    determineParents();
-    setNetWinding();
+    let output = featureCollection(outputFeatureArray) as FeatureCollection<
+      Polygon,
+      {
+        index: number;
+        parent: number;
+        winding: number;
+        netWinding: number | undefined;
+      }
+    >;
+    determineParents(output);
+    setNetWinding(output);
```

Comment:
`var output` was declared twice and the types disagreed, and then the determineParents and setNetWinding calls were referencing it quasi-implicitly because of the closure. Gnarly stuff.